examples/feature_selection/plot_compare_feature_selection.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
We will probably avoid to have a new example and instead we should edit an existing one.